packages/lib/filterProfileName.ts

```diff
@@ -0,0 +1,16 @@
+import { PROFILE_NAME_FILTER_REGEX } from 'data';
+
+/**
+ * Remove restricted symbols from profile name
+ *
+ * @param name Profile name
+ * @returns Profile name with restricted symbols removed
+ */
+const filterProfileName = (name: string | null | undefined): string | null => {
+  if (!name) {
+    return null;
+  }
+  return name.replace(PROFILE_NAME_FILTER_REGEX, ' ');
```

Comment:
```suggestion
  }

  return name.replace(PROFILE_NAME_FILTER_REGEX, ' ');
```

Reply:
Also add `.trim()` after replace!

Reply:
Ahh, true! Btw, I wasn't sure what character we want to replace with, so I went with a simple space.